Documentation/distributions.md

```diff
@@ -40,20 +49,21 @@ sudo apt-get install rkt
 Note that due to an outstanding bug (https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=823322) one has to use the "coreos" stage1 image:
 
 ```
-sudo rkt run --insecure-options=image --stage1-name=coreos.com/rkt/stage1-coreos:1.5.1 docker://nginx
+sudo rkt run --insecure-options=image --stage1-name=coreos.com/rkt/stage1-coreos:1.16.0 docker://nginx
 ```
 
-If you prefer to install the newest version of rkt follow the instructions in the Ubuntu section below.
+If you don't run sid, or wish for a newer version, you can [install manually](#deb-based).
 
 ## Fedora
 
-rkt is packaged in the development version of Fedora, [Rawhide](https://fedoraproject.org/wiki/Releases/Rawhide):
+Fedora version 24 and up include rkt, with some caveats. __Careful!__ The version of rkt in Fedora
```

Comment:
"Since Fedora version 24, rkt packages are available in the [main repository](https://apps.fedoraproject.org/packages/rkt). We recommend using recent Fedora releases or a [manually installed](#rpm-based) package in order to have an up-to-date rkt binary."